app/src/main/java/com/jnj/vaccinetracker/reportsoverview/vaccinesoverview/screens/VaccinesOverviewFragment.kt

```diff
@@ -303,35 +326,93 @@ class VaccinesOverviewFragment : BaseFragment(),
         }
     }
 
+
+    @SuppressLint("SetTextI18n")
     private fun applyFilters(
-        vaccinesData: List<VaccineObservationDTO> = vaccinesOverviewViewModel.vaccineDTOs.value
-            ?: emptyList()
+        vaccinesData: List<VaccineObservationDTO> = vaccinesOverviewViewModel.vaccineDTOs.value ?: emptyList()
     ) {
+        val isStatic = { loc: String? -> loc.equals("static", ignoreCase = true) }
+        val isOutreach = { loc: String? ->
+            loc?.let {
+                it.equals("outreach", ignoreCase = true) ||
+                        it.equals("outreach immunization", ignoreCase = true) ||
```

Comment:
are these options like "outreach immunization" or "outreach session" possible? do we have such options in mobile app?

Reply:
Resolved

Reply:
what is resolved? do these options are available in our application? "outreach immunization? "outreach session"?